packages/strapi-design-system/src/Select/MultiSelect.tsx

```diff
@@ -164,7 +170,7 @@ export const MultiSelect = ({
         >
           <SelectParts.Trigger
             ref={triggerRef}
-            aria-label={label}
+            aria-label={label ?? ariaLabel}
```

Comment:
Should this be the other way around, so that `ariaLabel` takes precedence over label in case both are passed in?